### ISSUE TITLE
Problème de sauvegarde du style dans la carte

### ISSUE THREAD

**Issue by opetitprestataire** [IGNF-Ma-carte/mcviewer#1]
Bonjour,

Je ne parviens pas à créer une carte statistique dans l'outil avec le fichier suivant :
[indices_commune_3.csv](https://github.com/user-attachments/files/17132140/indices_commune_3.csv)
Je paramètre ma carte comme suit :
![image](https://github.com/user-attachments/assets/ba0f202b-28c6-4ddf-b526-d96baecee34f)
Cependant en mode carte je perds ma symbologie et si j'essaie de passer par le biais du SLD j'ai l'erreur suivante :
![image](https://github.com/user-attachments/assets/7b853126-8640-4dc7-a20b-2cbde404b04d)

**Comment by Viglino** [comment]
Effectivement, il semble qu'il y ait un bug dans le chargement des statistiques en mode sectoriel.
Nous regardons d'où vient le problème.

**Comment by Viglino** [comment]
Le fichier (csv) fourni contient des cellules qui ne sont pas renseignées.
![image](https://github.com/user-attachments/assets/2efa8e2a-4df8-404e-b5b7-1c98797a4d7c)
En remplissant les cellules vides par 0, cela marche : [indices_commune_3.csv](https://github.com/user-attachments/files/17145007/indices_commune_3.csv)

Un fix [a été publié](https://github.com/IGNF-Ma-carte/mcutils/commit/ebd35531bb6fd5a8bc62792ad496923d9491a987) pour faire la transformation à la volée dans la prochaine version de l'outil.

**Comment by opetitprestataire** [comment]
Merci beaucoup !
